src/patternfly/components/NotificationBadge/notification-badge.scss

```diff
@@ -1,35 +1,48 @@
 .pf-c-notification-badge {
-  --pf-c-notification-badge--after--BorderColor: var(--pf-global--BackgroundColor--100);
-  --pf-c-notification-badge--after--BorderRadius: var(--pf-global--BorderRadius--lg);
+  --pf-c-notification-badge--Color: var(--pf-global--Color--light-100);
+
+  // update all --after variables to --before at breaking change
+  // ::after border treatment
+  --pf-c-notification-badge--after--BorderColor: var(--pf-global--BackgroundColor--dark-200);
+  --pf-c-notification-badge--after--BorderRadius: var(--pf-global--BorderRadius--sm);
   --pf-c-notification-badge--after--BorderWidth: var(--pf-global--BorderWidth--sm);
-  --pf-c-notification-badge--after--Top: 0;
-  --pf-c-notification-badge--after--Right: 0;
-  --pf-c-notification-badge--after--Width: calc(var(--pf-global--spacer--sm) + var(--pf-c-notification-badge--after--BorderWidth) + var(--pf-c-notification-badge--after--BorderWidth));
-  --pf-c-notification-badge--after--Height: calc(var(--pf-global--spacer--sm) + var(--pf-c-notification-badge--after--BorderWidth) + var(--pf-c-notification-badge--after--BorderWidth));
+  --pf-c-notification-badge--after--Top: 0; // remove at breaking change
```

Comment:
won't you need either `top: 0; right: 0;` or `width: 100%; height: 100%;` for the pseudo element to consume the parent? looks like these are all marked for removal.

Reply:
I mean, remove the variable and replace w/0

Reply:
Updated the comment here: https://github.com/patternfly/patternfly/pull/3284/files#diff-cb09d885cedbef17bb0765b2432406b9R39